analyzers/rspec/cs/S6422_c#.html

```diff
@@ -0,0 +1,73 @@
+<p>Making blocking calls to <code>async</code> methods transforms something that was intended to be asynchronous into a synchronous block. Doing so
+inside an Azure Function can lead to thread exhaustion.</p>
```

Comment:
>lead to thread exhaustion

Is this backed by some research? Azure Function may not be affected by that or the mitigation strategies are different in that environment.

Reply:
I have no idea, let's ask Andrei on slack